tools/tf_env_collect.sh

```diff
@@ -29,14 +29,29 @@ die() {
 echo "Collecting system information..."
 
 OUTPUT_FILE=tf_env.txt
-python_bin_path=$(which python || which python3 || die "Cannot find Python binary")
+PYTHON_BIN_PATH="$(which python || which python3 || die "Cannot find Python binary")"
+
+HEADER_WIDTH=68
+# Create a string of HEADER_WIDTH "=" characters
+HEADER=$(printf "%*s" "$HEADER_WIDTH" "" | sed 's/ /=/g')
+TITLE="GETTING STARTED"
+
+print_header () {
+  local TITLE="$1"
+  echo
+  # Echo to STDOUT
+  echo "== $TITLE ${HEADER:${#TITLE}+4}"
+  # Echo to STDERR (to show progress to the user as it runs)
+  echo "== $TITLE ${HEADER:${#TITLE}+4}" 1>&2
+}
+
+# Clear the output file
+echo > "$OUTPUT_FILE"
 
 {
-echo
-echo '== check python ==================================================='
-} >> ${OUTPUT_FILE}
+  print_header "check python"
 
-cat <<EOF > /tmp/check_python.py
```

Comment:
I don't know why the last guy chose to make a bunch of python files. I'm pretty sure that he just didn't know the syntax for this.